.github/ISSUE_TEMPLATE/release-team-lead-progress.md

```diff
@@ -0,0 +1,271 @@
+
+#### With Code Freeze
+- As needed, assist the Bug Triage Lead and Enhancements Lead removing PRs and enhancements from the milestone that aren't merged in time `/milestone clear`
+
+#### After Code Freeze
+- Wait for exception requests and bring them to `#sig-release`
+- [ ] Verified that the [`release-x.y` branch](https://github.com/kubernetes/kubernetes/branches) has been automatically created at the start of Test Freeze
+- [ ] Verified that the Branch Manager created the CI board on [Testgrid](https://testgrid.k8s.io/sig-release) for the release cycle (1.XX-blocking & 1.XX-informing)
+<!-- - [ ] Prepare for x.y.0-rc.0 release (shortly after Test Freeze), ensuring x.y-blocking, master-blocking are 100% green, or all failures have issues filed and are being actively worked. -->
+- [ ] Published the "Removals, Deprecations, and Major Changes blog"
+
+### 6. Test Freeze up to Release Day (~Week 14)
+- [ ] Verified together with the release-docs team that all KEPs with required documentation are ready for review
+- [ ] Completed release theme (slogan, logo and explanation text) and add it to the [release cycle documentation in k/sig-release/releases/release-1.XX](https://github.com/kubernetes/sig-release/tree/master/releases)
+<!-- - [ ] Branch Manager ensures automation is actively enforcing merge blocking labeling and other release policies -->
+<!-- - [ ] The Release Team needs to look at any in-flight PRs and ensure nothing is being jammed in at the last minute without proper tests, review, etc. This is something to watch extremely closely because it happens every release. Just watch what gets merged closely after Code Freeze. Incorrectly merged items need assessed and perhaps reverted. GitHub has a query comparing a release branch and master. -->
+<!-- - [ ] Assist the Documentation Leads in collecting missing docs PRs. -->
+- [ ] Decided with the Release team if burndown meetings are necessary or updates are done via Slack thread on Tuesdays and Thursdays
+<!-- - [ ] Release notes and themes should be close to done if not completed. There is a script that gathers notes from PRs but it’s still in progress. As the lead, you may need to help assemble the notes. -->
+- [ ] Discussed Release Lead succession with the EA and sig-release leads
+<!-- - [ ] Identify potential vacancies on the incoming Release Team and begin asking team shadows and former team shadows to serve as Lead in the upcoming cycle. Continue to improve and uphold the Release Team Selection process. -->
+- [ ] The task is now to ensure the release branch is ready to go. This means there are zero pending PRs, no failing x.y-blocking tests, no open issues in the milestone. This will continue until release day.
```

Comment:
```suggestion
- [ ] The task is now to ensure the release branch is ready to go. This means there are zero pending PRs, no failing 1.XX-blocking tests, no open issues in the milestone. This will continue until release day.
```